tools/Dockerfile.CI

```diff
@@ -0,0 +1,34 @@
+FROM timbru31/java-node:8-slim
+
+# Install Bazel
+#  Reference: https://docs.bazel.build/versions/master/install-ubuntu.html)
+RUN echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" | tee /etc/apt/sources.list.d/bazel.list
+RUN curl https://bazel.build/bazel-release.pub.gpg | apt-key add -
+RUN apt-get update && apt-get install bazel -y
+
+
+# Install Chrome for Selenium
+#  Reference: https://jpmelos.com/articles/how-use-chrome-selenium-inside-docker-container-running-python/
+RUN curl https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb -o /chrome.deb
+RUN dpkg -i /chrome.deb || apt-get install -yf
+RUN rm /chrome.deb
+# Install chromedriver for Selenium
+RUN curl https://chromedriver.storage.googleapis.com/2.31/chromedriver_linux64.zip -o /usr/local/bin/chromedriver
+RUN chmod +x /usr/local/bin/chromedriver
+
+# Set up workspace
+RUN mkdir -p /usr/src/app
+ENV WORKSPACE /usr/src/app
+WORKDIR /usr/src/app
+
+
+# Install programs for runtime use (e.g. by Bazel, Node-GYP)
+RUN apt-get update && apt-get upgrade -y && \
+    apt-get install bash git python g++ make -y
```

Comment:
They recommend putting each package on its own line and sorting alphabetically: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#sort-multi-line-arguments

```
apt-get install -y \
    bash \
    g++ \
    git \
    make \
    python
```